web/src/components/feed/PostComments/index.tsx

```diff
@@ -109,11 +109,12 @@ export function PostComment({
         (comment.privacy == PrivacyType.PRIVATE
           ? ' card-notification--comment-private'
           : '') +
-        (isReply ? ' card-notification--reply' : '')
+        (isReply ? ' card-notification--reply' : '') + 
+        (sessionUser && (sessionUser.id == comment.author.id) ? ' card-notification--reply' : '')
       }
     >
-      <Comment comment={comment} />
-      <div className="message__actions">
+      <Comment comment={comment} sessionUser={sessionUser}/>
+      <div className={'message__actions ' + (sessionUser && (sessionUser.id == comment.author.id) ? ' ' : 'message__actions--you') }>
```

Comment:
what is this doing?

Reply:
this is moving other users' replies to left side of the comments flow

Reply:
why now you are checking the sessionUser and the author id?